src/components/NewMovie/NewMovie.tsx

```diff
@@ -1,45 +1,91 @@
 import { useState } from 'react';
 import { TextField } from '../TextField';
+import { Movie } from '../../types/Movie';
 
-export const NewMovie = () => {
-  // Increase the count after successful form submission
-  // to reset touched status of all the `Field`s
-  const [count] = useState(0);
+// Increase the count after successful form submission
+// to reset touched status of all the `Field`s
+const initialValue = {
+  title: '',
+  description: '',
+  imgUrl: '',
+  imdbUrl: '',
+  imdbId: '',
+};
+
+type Props = {
+  onAdd:(movie:Movie) => void;
+};
+
+export const NewMovie:React.FC<Props> = ({ onAdd }) => {
+  const [count, setCount] = useState(0);
+  const [textAtributes, setTextAtributes] = useState(initialValue);
+
+  const addOne = () => {
```

Comment:
The current name of the function does not answer the question "what does it do", but raises new questions :)
```suggestion
  const incrementCounter = () => {
```